### PR TITLE
feat: SQL Server script enhancements

### PR DESCRIPTION
**Pull request by shane-borden**

### CODE REVIEW

File: docs/user_guide/sqlserver/collection_scripts.md [hunk 1]
```diff
@@ -92,44 +92,44 @@ In order to begin running the Database Migration Assessment Collection process,
 #### Perfmon Requirements (Optional)
 
 - If you have your own perfmon counters capturing the following statistics or run on a SQL Server Product such as Amazon RDS or Google CloudSQL for SQL Server, skip to step b, otherwise proceed to step a.
-** The Perfmon data collection process is optional and can be safely skipped.  However, there will be no right sizing information in the assessment report.
+  \*\* The Perfmon data collection process is optional and can be safely skipped. However, there will be no right sizing information in the assessment report.
   <br/>
```
Comment (cofin) [inline docs/user_guide/sqlserver/collection_scripts.md:96]: Is the HTML break supposed to be here?

File: scripts/collector/sqlserver/dmaCollectorCommonFunctions.psm1 [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# See the License for the specific language governing permissions and
+# limitations under the License.
+<#
+.SYNOPSIS
+    .
+.DESCRIPTION
+    Creates a user within the SQL Server Database using "Windows Authentication" with the necessary permissions 
+    needed to execute subsequent scripts to collect data from SQL Server and Perfmon to be uploaded to Google Database Migration Assistant for review.
+
+    If user and password are supplied, that will be used to execute the script.  Otherwise default credentials hardcoded in the script will be used
```
Comment (cofin) [inline scripts/collector/sqlserver/dmaCollectorCommonFunctions.psm1:21]: I think we can just remove this line?  There are no default creds anymore, correct?

File: docs/user_guide/sqlserver/collection_scripts.md (hunk 1 above)
Comment (shane-borden) [reply docs/user_guide/sqlserver/collection_scripts.md:96]: Yes.  Needed to add it once I checked the markdown in vscode.

File: scripts/collector/sqlserver/dmaCollectorCommonFunctions.psm1 (hunk 2 above)
Comment (shane-borden) [reply scripts/collector/sqlserver/dmaCollectorCommonFunctions.psm1:21]: Correct.  Looks like I missed this edit.

File: docs/user_guide/sqlserver/collection_scripts.md (hunk 1 above)
Comment (cofin) [inline docs/user_guide/sqlserver/collection_scripts.md:96]:
```suggestion
```

File: scripts/collector/sqlserver/dmaCollectorCommonFunctions.psm1 (hunk 2 above)
Comment (cofin) [inline scripts/collector/sqlserver/dmaCollectorCommonFunctions.psm1:21]:
```suggestion
```

File: docs/user_guide/sqlserver/collection_scripts.md [hunk 3]
```diff
@@ -92,44 +92,44 @@ In order to begin running the Database Migration Assessment Collection process,
...
+          \SQLServer:Buffer Manager\Page lookups/sec
+          \SQLServer:Buffer Manager\Page reads/sec
+          \SQLServer:Buffer Manager\Page writes/sec
+          \SQLServer:General Statistics\User Connections
+          \SQLServer:Memory Manager\Memory Grants Pending
+          \SQLServer:Memory Manager\Target Server Memory (KB)
+          \SQLServer:Memory Manager\Total Server Memory (KB)
+          \SQLServer:SQL Statistics\Batch Requests/sec
+
+      <br/>
```
Comment (cofin) [inline docs/user_guide/sqlserver/collection_scripts.md:122]:
```suggestion
```

File: docs/user_guide/sqlserver/collection_scripts.md (hunk 1 above)
Comment (shane-borden) [reply docs/user_guide/sqlserver/collection_scripts.md:96]: This change needs to stay.

**Review by cofin — APPROVED** [review]
LGTM other than the 2 comments

**Review by cofin — APPROVED** [review]

**Pull request merged**